tests/comments/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -1317,7 +1317,7 @@ if (/* Comments */ true /* Comments */) {
 } else {
 }
 
-if (2 /* MB_OVERLOAD_STRING */ & (int) ini_get('mbstring.func_overload')) {
+if (2 /* MB_OVERLOAD_STRING */ & ((int) ini_get('mbstring.func_overload'))) {
```

Comment:
Expected. Look on this `if ((int) $a > $b) {}`
Very meas leading too.
- `if (((int) $a) > $b) {}`
- `if ((int) ($a > $b)) {}`

Reply:
Also `(` and `)` add readability in conditions.